.github/dependabot.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+updates:
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: "22:00"
```

Comment:
I think that in order to test this today we'd have to change the hour.

Otherwise, I think that time interval is just fine.